zebra-chain/src/sapling/tree.rs

```diff
@@ -115,12 +115,6 @@ struct NoteCommitmentTree {
     count: u32,
 }
 
-impl From<Vec<NoteCommitment>> for NoteCommitmentTree {
-    fn from(_values: Vec<NoteCommitment>) -> Self {
-        unimplemented!();
-    }
-}
```

Comment:
FYI this is PR #2407 which should merge soon.

Reply:
Thinking about merge conflicts here: we probably just want to let PR #2407 deal with this.